schema/cassandra/cadence/versioned/v0.12/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Can we do this in v0.11?  So we can roll this out in next release.

Reply:
OK will do that.